roles/repository/molecule/default/converge.yml

```diff
@@ -33,16 +33,22 @@
             version: 3.5.0
         amp_downloads:
           - url: "{{ nexus_repository.enterprise_releases }}org/alfresco/integrations/alfresco-googledrive-repo-enterprise/{{ amps.googledrive_repo.version }}/alfresco-googledrive-repo-enterprise-{{ amps.googledrive_repo.version }}.amp"
+            sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}org/alfresco/integrations/alfresco-googledrive-repo-enterprise/{{ amps.googledrive_repo.version }}/alfresco-googledrive-repo-enterprise-{{ amps.googledrive_repo.version }}.amp.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
             dest: "{{ content_folder }}/amps_repo/alfresco-googledrive-repo-enterprise.amp"
-          - url: "https://nexus.alfresco.com/nexus/service/local/repo_groups/public/content/org/alfresco/integrations/alfresco-googledrive-share/{{ amps.googledrive_share.version }}/alfresco-googledrive-share-{{ amps.googledrive_share.version }}.amp"
+          - url: "{{ nexus_repository.releases }}org/alfresco/integrations/alfresco-googledrive-share/{{ amps.googledrive_share.version }}/alfresco-googledrive-share-{{ amps.googledrive_share.version }}.amp"
+            sha1_checksum: "{{ lookup('url', '{{ nexus_repository.releases }}org/alfresco/integrations/alfresco-googledrive-share/{{ amps.googledrive_share.version }}/alfresco-googledrive-share-{{ amps.googledrive_share.version }}.amp.sha1') }}"
             dest: "{{ content_folder }}/amps_share/alfresco-googledrive-share.amp"
           - url: "{{ nexus_repository.enterprise_releases }}org/alfresco/services/sync/alfresco-device-sync-repo/{{ amps.device_sync.version }}/alfresco-device-sync-repo-{{ amps.device_sync.version }}.amp"
+            sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}org/alfresco/services/sync/alfresco-device-sync-repo/{{ amps.device_sync.version }}/alfresco-device-sync-repo-{{ amps.device_sync.version }}.amp.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
             dest: "{{ content_folder }}/amps_repo/alfresco-device-sync-repo.amp"
           - url: "{{ nexus_repository.releases }}org/alfresco/aos-module/alfresco-aos-module/{{ amps.aos_module.version }}/alfresco-aos-module-{{ amps.aos_module.version }}.amp"
+            sha1_checksum: "{{ lookup('url', '{{ nexus_repository.releases }}org/alfresco/aos-module/alfresco-aos-module/{{ amps.aos_module.version }}/alfresco-aos-module-{{ amps.aos_module.version }}.amp.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
```

Comment:
Looks like you missed this one (nexus credentials) or maybe I did in the first review ;-)

Reply:
@gavincornwell I don't know github does this, but if you check the file, you'll see that this is already changed. The file:

https://github.com/Alfresco/alfresco-ansible-deployment/blob/OPSEXP-783-flatvars/roles/repository/molecule/default/converge.yml

Reply:
Hmm strange, this has been happening a few times recently, especially with images, GitHub bug maybe!?

If they are all fixed then ignore me!

Reply:
Not all are fixed though.

Reply:
Ah, OK.